src/coreclr/jit/hwintrinsic.h

```diff
@@ -657,13 +656,7 @@ struct HWIntrinsicInfo
     static bool RequiresCodegen(NamedIntrinsic id)
     {
         HWIntrinsicFlag flags = lookupFlags(id);
-        return (flags & HW_Flag_NoCodeGen) == 0;
-    }
-
-    static bool GeneratesMultipleIns(NamedIntrinsic id)
```

Comment:
How are we handling `MultiIns` if we removed the flag?

Reply:
Via `SpecialCodeGen` as that's all the flag was being used for today. It was basically a flag that wasn't actually needed.